samples/bluetooth/mesh/dfu/distributor/sysbuild/mcuboot/boards/nrf54l15dk_nrf54l15_cpuapp_ext_flash.overlay

```diff
@@ -0,0 +1,48 @@
+/ {
+	chosen {
+		nordic,pm-ext-flash = &mx25r64;
+		zephyr,code-partition = &boot_partition;
+	};
+};
+
+/delete-node/ &boot_partition;
+/delete-node/ &slot0_partition;
+/delete-node/ &slot1_partition;
+
+/delete-node/ &storage_partition;
+
+&cpuapp_rram {
+	reg = < 0x0 DT_SIZE_K(1524) >;
+
+	partitions {
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x000000000 0x00014000>;
+		};
+
+		slot0_partition: partition@14000 {
+			label = "image-0";
+			reg = <0x000014000 0x0015A000>;
```

Comment:
Did you check that these partitions addresses and sizes coincide to what Partition Manager generates for the sample?

Reply:
Hm.... but we do not use static Partition Manager configuration. When sample is built, Partition Manager knows only the sequence of areas but not exact addresses. Could you check it any case?

Reply:
But you might have a point, even though this file is from sdk-mcuboot, we never included it before because we never build `_ext_flash` on this board.

Reply:
Checked now on the desk. This part of the overlay is not used by the sample since the sample uses partition manager. I'd prefer to keep it so this file stays as a copy of the sdk-mcuboot overlay file, even though this sample in particular doesn't need exactly this part. If we want to, there are probably several not-needed parts in these copied files that we could clean up, but I suggest a separate task in that case, since it requires some research into configuring mcuboot.

Reply:
Tested correctly now, conclusion still stands. Sorry for confusion.